src/modules/utils/path.rs

```diff
@@ -80,6 +82,11 @@ impl PathExt for Path {
         let (_, path) = normalize::normalize_path(self);
         path
     }
+
+    fn device_id(&self) -> u64 {
+        // Maybe it should us unimplemented!
```

Comment:
Typo:
```suggestion
        // Maybe it should be unimplemented!
```

I wonder if [std::os::windows::fs::MetadataExt::volume_serial_number](https://doc.rust-lang.org/std/os/windows/fs/trait.MetadataExt.html#tymethod.volume_serial_number) will ever get stabilized

Reply:
In fact I meant that it might use `unimplemented!` and do the `cfg` stuffs needed to still be usable under windows.

But indeed I looked at `volume_serial_number` and that's what we need but the last comment of the linked issue did not made me hopeful.